binding/identity.go

```diff
@@ -7,45 +7,65 @@ import (
 //
 // Identity API.
 type Identity struct {
-	// hub API client.
-	Client *Client
+	client *Client
 }
 
 //
 // Create a Identity.
 func (h *Identity) Create(r *api.Identity) (err error) {
-	err = h.Client.Post(api.IdentitiesRoot, &r)
+	err = h.client.Post(api.IdentitiesRoot, &r)
 	return
 }
 
 //
 // Get a Identity by ID.
 func (h *Identity) Get(id uint) (r *api.Identity, err error) {
 	r = &api.Identity{}
+	p := Param{
+		Key:   api.Decrypted,
+		Value: "1",
+	}
```

Comment:
Adding the _decrypted_ parameter.  This assumes that for binding users, there is no use case for fetching encrypted identities.